### DISCUSSION TITLE
Support for the Ascota Line Of Computers

### DISCUSSION THREAD

**Discussion by mmoeller86** [86Box/86Box#3334] [Ideas]
I request support for the Ascota Line of Computers.
They were had 80286 or 80386 Processors.
There maybe existed Models with higher (486+) Processors.

I read it on the main Robotron Page: https://www.robotrontechnik.de/index.htm?/html/computer/ascotapc.htm

**Comment by OBattler** [comment]
Same as on your other request - we would need BIOS'es, motherboard pictures, and documentation.

**Comment by mmoeller86** [comment]
I don't have BIOS'es, Motherboard Pictures are maybe in the Web and Documentation hmm....

I maybe ask at https://www.robotrontechnik.de/html/forum/thwb/index.php

Greets,
Martin

**Comment by mmoeller86** [comment]
Are you registered at this forum ?
If not I can make it for you !

**Comment by mmoeller86** [comment]
I tried to register at this forum, but there actually came no answer !

**Comment by mmoeller86** [comment]
If I had the BIOS for the EC 1835 Turbo, we can start with a generic clone and could
test the BIOS.

**Comment by mmoeller86** [comment]
And then enhance to a 80386-based computer.

**Comment by mmoeller86** [comment]
We could start with a Award or AMI 386 Clone...

**Comment by mmoeller86** [comment]
If I'm registered I will ask for the BIOS'es, Floppy Images, Documentation, Motherboard Images and so on.
And if I have it will post a Link for the Files.

**Comment by mmoeller86** [comment]
I will first get the files stated above and will then start a generic clone from
either Award 386 (preferred) or AMI 386 (sub-preferred).

**Comment by mmoeller86** [comment]
I will tune it until the BIOS boots and make the POST and then will probably boot
with the Award 386 or AMI 386 generic clone.

**Comment by mmoeller86** [comment]
For the Ascota Computers which maybe use more modern Processors like the 486 or
Intel Pentium, which maybe existed, is a special activity. Lets see what I can use
as base.

The same for the Soemtron Computers.
